main.go

```diff
@@ -194,5 +21,9 @@ func main() {
 	door.SetOpen(false)
 	door.SetLocked(true)
 	motor.SetOn(true)
+	motor.SetRPM(2)
+	wheels.SetHalfMotorRPM(motor)
```

Comment:
@lisetterz This method is not very extensible.  Perhaps a function which in on the car, and takes a value as input for the c.WheelManipulator.SetRPM function.

Reply:
I believe you mean something more like this 8224910427e38c24dbaeaf9c11f176001d50cc52 :)